test/jdk/java/foreign/TestHandshake.java

```diff
@@ -56,6 +56,8 @@
     static final int MAX_EXECUTOR_WAIT_SECONDS = 10;
     static final int MAX_THREAD_SPIN_WAIT_MILLIS = 200;
 
+    static final int NUM_ACCESSORS = Math.max(10, Runtime.getRuntime().availableProcessors());
+
```

Comment:
`min` rather than `max` so as to clamp at a maximum of 10 accessors?

Reply:
Whoops - of course :-)